components/breadcrumb/examples/00-default.js

```diff
@@ -10,16 +10,25 @@ function Example({ brand }) {
 		<GEL brand={brand}>
 			<Intopia />
 
-			<h2>Declarative as links</h2>
+			<h2>Default</h2>
+			<Breadcrumb>
+				<Crumb text="Home" />
+				<Crumb text="Personal" />
+				<Crumb text="Credit cards" />
+			</Breadcrumb>
+
+			<hr />
+
+			<h2>Declarative</h2>
+
+			<h3>As links</h3>
 			<Breadcrumb>
 				<Crumb href="#/" text="Home" />
 				<Crumb href="#/personal-banking/" text="Personal" />
 				<Crumb href="#/credit-cards/" text="Credit cards" />
 			</Breadcrumb>
 
-			<hr />
-
-			<h2>Declarative as router</h2>
```

Comment:
It would be good to see the `tag` prop in the examples somewhere

Reply:
yes, could I get either you @dominikwilkowski or you @jaortiz to assist?